tests/unit/db/test_datastore.py

```diff
@@ -0,0 +1,102 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2020 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import datetime
+import textwrap
+from unittest.mock import Mock
+
+import pytest
+import tinydb
+
+from snapcraft.internal.db import datastore, errors, migration
+
+
+@pytest.fixture(autouse=True)
+def mock_snapcraft_version(monkeypatch):
+    monkeypatch.setattr(migration.snapcraft, "__version__", 42)
+
+
+@pytest.fixture(autouse=True)
+def mock_datetime(monkeypatch):
+    datetime_mock = Mock(wraps=datetime.datetime)
+    datetime_mock.now.return_value = datetime.datetime(2020, 1, 2, 3, 4, 5, 6)
+    monkeypatch.setattr(migration, "datetime", datetime_mock)
+
+
+def test_get_snapcraft_version(monkeypatch):
```

Comment:
is there a need for monkeypatch here? I might have misunderstood autouse

Reply:
Probably an artifact of previous implementation, will remove.

Reply:
fixed (removed altogether)